components/toasts/SwapSuccess.tsx

```diff
@@ -0,0 +1,86 @@
+import { CloseIcon, InfoIcon, WarningIcon } from "@chakra-ui/icons";
+import { Button, Flex, Text } from "@chakra-ui/react";
+import {
+  Id,
+  toast,
+  ToastContentProps,
+  ToastOptions,
+  TypeOptions,
+} from "react-toastify";
+import CheckIcon from "public/icons/check.svg";
+
+interface BodyProps {
+  mainText: string;
+  type?: TypeOptions | null | undefined;
```

Comment:
if it is optional I guess we don't need | null | undefined

Reply:
Sorry, this is the only way I could get it to work! ':D